github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```
@@ -114,12 +119,12 @@ presubmits:
     annotations:
       fork-per-release: "true"
       testgrid-dashboards: kubevirt-presubmits
-    always_run: false
-    optional: true
+    always_run: true
+    optional: false
```

Let's run this job in the new cluster
```suggestion
    optional: false
    cluster: prow-workloads
```